### ISSUE TITLE
Fuel Scoop not functioning

### ISSUE THREAD

**Issue by GespenstR** [pioneerspacesim/pioneer#5237]
### Observed behaviour
![screenshot-20210813-131828](https://user-images.githubusercontent.com/88902910/129399940-8400fa83-3bd5-4fbf-871b-8493f881ac08.png)
While flying around NN 4247 b, I tried to get close to the gas giant to get fuel, but despite being withing the parameters stated on the wiki, nothing happened. When I tried to get closer, nothing happened.

### Expected behaviour
The fuel scoop messages would have shown up during flight, and the ship would be stocked up with Hydrogen.

### Steps to reproduce
See save file here:
https://drive.google.com/file/d/1Lo2vJxyTBYQDKNXZ8wxa7a__RuRqlOxW/view?usp=sharing

I placed the ship right next to the planet in question

__My pioneer version (and OS):__
Info: ver 20210723 (0b82c65) on: Linux

**Comment by joonicks** [comment]
your speed needs to be in excess of 1000m/s before it starts scooping, you seem to be just on the edge of that

**Comment by nozmajner** [comment]
I just checked that with a Mola save from discord. It turns out the Mola is struggling to fly at 1km/s in 1 atm pressure. It can be done for very short periods of time though
![screenshot-20210923-220552](https://user-images.githubusercontent.com/4182678/134577241-64b51fb0-15f0-4025-a534-a427150b5589.png)
.

**Comment by GespenstR** [comment]
I think a significant part of the issue was in regards to information about the Fuel Scoop itself. When I opened this ticket, the Wiki states that the requirements were 1km and .10atm. Naturally this lead to confusion when trying the scoop out. This has since been changed to better reflect the current build of the game.

I tried the scoop out again recently, and have found that it seems to work properly at around 1atm rather than the previous .10atm. This also gets you somewhat close the the planet's surface, about 200km above usually.
